sql/donut.sql

```diff
@@ -139,6 +138,13 @@ CREATE TABLE positions (
     group_id INT         NOT NULL,
     pos_id   INT         NOT NULL AUTO_INCREMENT,
     pos_name VARCHAR(32) NOT NULL,
+    send        BOOLEAN DEFAULT FALSE, -- Toggles whether or not this position
```

Comment:
send, control, and receive have their defaults defined here, but also need to be changed on a person by person basis.